addons/website/static/tests/tours/website_update_column_count.js

```diff
@@ -116,3 +117,93 @@ wTourUtils.clickOnSnippet({
+            + ":has(.order-lg-0[style*='order: 2;']:nth-child(3))",
+        isCheck: true,
+    },
+    // Add a mobile order to the "Text-Image" snippet columns.
+    {
+        content: "Click on the first column of the 'Text-Image' snippet",
+        trigger: `${textImageSnippetRow} > div:nth-child(1)`,
+    }, {
+        content: "Change the orders of the 1st and 2nd columns",
+        trigger: "iframe .o_overlay_move_options [data-name='move_right_opt']",
+    }, {
+        content: "Check that the mobile order classes and styles are correct",
+        trigger: `${textImageSnippetRow}:has(.order-lg-0[style*='order: 1;']:nth-child(1))`
+            + ":has(.order-lg-0[style*='order: 0;']:nth-child(2))",
+        isCheck: true,
+    },
+    // Test the drag and drop in the same snippet.
+    {
+        content: "Toggle the desktop view",
+        trigger: ".o_we_website_top_actions [data-action='mobile']",
+    }, {
+        content: "Drag a 'Text-Image' column and drop it in the same snippet",
+        trigger: "iframe .o_overlay_move_options .o_move_handle",
+        run: `drag_and_drop_native ${textImageSnippetRow}`,
+    }, {
+        content: "Check that the mobile order classes and styles were removed",
+        trigger: `${textImageSnippetRow}:not(:has(.order-lg-0[style*='order: ']))`,
+        isCheck: true,
+    },
+    // Add again a mobile order to the "Text-Image" snippet columns.
+    {
+        content: "Toggle the mobile preview",
+        trigger: ".o_we_website_top_actions [data-action='mobile']",
+    }, {
+        content: "Click on the first column of the 'Text-Image' snippet",
+        trigger: `${textImageSnippetRow} > div:nth-child(1)`,
+    }, {
+        content: "Change the orders of the 1st and 2nd columns",
+        trigger: "iframe .o_overlay_move_options [data-name='move_right_opt']",
+    }, {
+        content: "Check that the mobile order classes and styles are correct",
+        trigger: `${textImageSnippetRow}:has(.order-lg-0[style*='order: 1;']:nth-child(1))`
+            + ":has(.order-lg-0[style*='order: 0;']:nth-child(2))",
+        isCheck: true,
+    },
+    // Test the drag and drop from "Columns" to "Text-Image".
+    {
+        content: "Toggle the desktop view",
+        trigger: ".o_we_website_top_actions [data-action='mobile']",
+    }, {
+        content: "Click on the second column of the 'Columns' snippet",
+        trigger: `${columnsSnippetRow} > div:nth-child(2)`,
+    }, {
+        content: "Drag the second column of 'Columns' and drop it in 'Text-Image'",
+        trigger: "iframe .o_overlay_move_options .o_move_handle",
+        run: `drag_and_drop_native ${textImageSnippetRow}`,
+    }, {
+        content: "Check that the mobile order classes and styles were removed in 'Text-Image'",
+        trigger: `${textImageSnippetRow}:not(:has(.order-lg-0[style*='order: ']))`,
+        isCheck: true,
+    }, {
+        content: "Check that the order gap left in 'Columns' was filled",
```

Comment:
Not sure to understand, what other task are you talking about ? It has already been done in the mentioned commit (which was merged, so it is not a prevision) :thinking:

Reply:
Ah yes of course, that's already how it behaves, but I thought you did that in prevision of task-3744636 (since the order right now doesn't do anything except prevent crashes, as the snippet is switched to a grid).

Reply:
Nevermind :facepalm: